app/api/dao/user.py

```diff
@@ -129,7 +130,9 @@ def update_user_profile(user_id, data):
             user.resume_url = data['resume_url']
 
         if 'photo_url' in data and data['photo_url']:
```

Comment:
not pythonic, a better way is:
try:
image_type = ...
except KeyError:
<handle>

Reply:
@ramitsawhney27  This practice is followed in whole function, Should I open an issue and fix it in another PR ?

Reply:
Lets just fix it here for now

Reply:
Although I haven't touched that line. If I'll do the changes it won't be a unnecessary change for the PR right. @ramitsawhney27

Reply:
That's not an issue.